rest/models/contracts.go

```diff
@@ -0,0 +1,161 @@
+package models
+
+// GetOptionsContract is the set of parameters for the GetOptionsContract method.
+type GetOptionsContractParams struct {
+	// Return the contract that contains this options ticker.
+	OptionsTicker string `validate:"required" path:"options_ticker"`
+
+	// Specify a point in time for the contract as of this date.
+	AsOf *Date `query:"as_of"`
+}
+
+func (p GetOptionsContractParams) WithDate(q Date) *GetOptionsContractParams {
+	p.AsOf = &q
+	return &p
+}
+
+// GetOptionsContractResponse is the response returned by the GetOptionsContract method.
+type GetOptionsContractResponse struct {
+	BaseResponse
+
+	// Options Contract results.
+	Results OptionsContract `json:"results,omitempty"`
```

Comment:
i decided not to document the response fields right now so we should remove this (and the line above) to be consistent